### PR TITLE
Add rest of supporting files for ScriptManager

### PR DESCRIPTION
**Pull request by twsouthwick**
Part of #9

### CODE REVIEW

**Comment by twsouthwick** [comment]
Once this and #136 is in, the ajax.aspx sample works :)

File: src/Extensions/Extensions.csproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <Nullable>disable</Nullable>
     <AssemblyName>WebForms.Extensions</AssemblyName>
     <RootNamespace>System.Web</RootNamespace>
+    <NoWarn>$(NoWarn);CA1823;CA1852;CA1822;CA2249;CA1845;CA1305;CS0649;CS0414;CA1805;CA1810;CA1846;CS0169</NoWarn>
```
Comment (twsouthwick) [inline src/Extensions/Extensions.csproj:10]: https://github.com/CoreWebForms/CoreWebForms/issues/138 tracks cleaning this up

**Review by birojnayak — APPROVED** [review]
clean up the header comments

File: src/Extensions/Globalization/ClientCultureInfo.cs [hunk 2]
```diff
@@ -0,0 +1,171 @@
+//------------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (twsouthwick) [reply src/Extensions/Globalization/ClientCultureInfo.cs]: done

**Pull request merged**